### DISCUSSION TITLE
WUT lists user-serviceable?

### DISCUSSION THREAD

**Discussion by blakesphere** [Stellarium/stellarium#1583] [Q&A]
Hi. 

I am really enjoying learning all the capabilities of the Astronomical Calculations feature. Amazing!

I have a question about the WUT tab.

Who made the category lists in the What's Up Tonight? Who made the Messier objects, the Plutinos, the Bright double stars? 

More to the point: Can ANYONE make them? Could I make my own? For example, I like viewing colour red stars. I have a number of red star candidates from lists on the interwebs. It'd be cool if I could make my own category for the Stellarium WUT. I'm happy hacking INI files, etc, so I'm fine with doing some work behind the scenes to make it happen...

cheers,
Blake

**Comment by alex-w** [comment]
Right now only developers can create lists for WUT tool. Theoretically we can add tool for creating custom lists of objects for WUT tool, but I see problem with unnamed stars.

P.S. All lists for WUT was made developers by filtering objects.

**Reply by blakesphere** [reply]
Understood. Thanks for the note.

In the meantime, I'm using bookmarks to make observing plan lists.

Thank you!

Blake

**Reply by alex-w** [reply]
I think we can add 2 special lists of stars (coloured doubles and carbon stars) + include the scripts-excursions for both lists.

**Reply by alex-w** [reply]
OK, carbon stars filter was added for Search and AstroCalc tools.

**Reply by alex-w** [reply]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot
